cmd/init.go

```diff
@@ -0,0 +1,556 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/amp-labs/cli/flags"
+	"github.com/amp-labs/cli/logger"
+	"github.com/amp-labs/cli/openapi"
+	"github.com/amp-labs/cli/request"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+)
+
+var ErrBadInput = errors.New("bad input")
+
+const (
+	NumProvidersShown = 10
+	YamlFileMode      = 0o644
+)
+
+func nonEmpty(fieldName string) func(string) error {
+	return func(s string) error {
+		if s == "" {
+			return fmt.Errorf("%w: %s cannot be empty", ErrBadInput, fieldName)
+		}
+
+		return nil
+	}
+}
+
+var initCmd = &cobra.Command{ //nolint:gochecknoglobals
+	Use:   "init",
+	Short: "create a new Ampersand project",
+	Long:  "Create a new Ampersand project.",
+	Run: func(cmd *cobra.Command, args []string) {
+		name, err := promptString("Name your integration", nonEmpty("Integration name"))
+		if err != nil {
+			logger.FatalErr("Unable to prompt for integration name", err)
+		}
+
+		provider, err := selectProvider(cmd.Context())
+		if err != nil {
+			logger.FatalErr("Unable to select provider", err)
+		}
+
+		disp := "my integration"
+
+		var integ openapi.Integration
+		integ.Name = name
+		integ.DisplayName = &disp
+		integ.Provider = provider.Name
+
```

Comment:
can you modify the order of these so it's:
- read
- write
- proxy

(that also follows the order we use in the docs, and we should be urging people to use read/write more than proxy)